pytorch_lightning/accelerators/legacy/cpu_accelerator.py

```diff
@@ -80,10 +83,15 @@ def test_step(self, args):
     def predict(self, args):
         return self._step(self.trainer.model.predict, args)
 
-    def sync_tensor(self,
-                    tensor: Union[torch.Tensor],
-                    group: Optional[Any] = None,
-                    reduce_op: Optional[Union[ReduceOp, str]] = None) -> torch.Tensor:
+    def to_device(self, batch):
+        return self.batch_to_device(batch)
```

Comment:
shouldn't it follow the superclass interface of batch_to_device?

Reply:
sorry, didn't get it. It calls batch_to_device from the `Accelerator` class only

Reply:
so far we did not call the transfer hook when transferring data from cpu to cpu, so we need to think here. what would be the best option?

Reply:
what I can do is allow only `on_before_batch_transfer` hook available for both CPU and DP. This way we can have `on_before_batch_transfer` available for all accelerators and won't be doing any `transfer_batch_to_device` on CPU. WDYT?

Reply:
ok, sounds good. 
If people want to do some sort of data augmentations, we should run the hook no matter the device or training mode. 
Even the transfer_batch_to_device hook, imo, should run on CPU too.